domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandling/Behandlingsinformasjon.kt

```diff
@@ -225,60 +225,60 @@ data class Behandlingsinformasjon(
     }
 
     data class Bosituasjon(
-        val delerBolig: Boolean,
-        val delerBoligMed: Boforhold.DelerBoligMed?,
-        val ektemakeEllerSamboerUnder67År: Boolean?,
+        val epsFnr: Fnr?,
+        val delerBolig: Boolean?,
         val ektemakeEllerSamboerUførFlyktning: Boolean?,
         val begrunnelse: String?
     ) : Base() {
         fun utledSats() = getBeregningStrategy().sats()
 
-        internal fun getBeregningStrategy() =
-            if (!delerBolig) {
-                BeregningStrategy.BorAlene
+        internal fun getBeregningStrategy(): BeregningStrategy {
+            if (epsFnr == null && (delerBolig != null && !delerBolig)) {
```

Comment:
Det har sikkert vært diskutert og er sikkert litt på siden, men man kan jo tenke seg at ektefellen bor i et annet land f.eks.? Kan de da ikke bo alene?

Reply:
Det skal ikke bli tatt hensyn til EPS hvis dem ikke bor sammen. Søker er derfor enslig / deler bolig med andre.

Reply:
Så dersom de ikke bor sammen, så fyller ikke saksbehandler ut epsFnr? Eller er det sånn at de er mutual exclusive (nor)? Dvs. en eller begge må være null?

Reply:
Fra det jeg har spurt og fått med meg, så ja, bare en av dem kan bli brukt om gangen. 

Jeg vil generelt kunne si at man kan rent teoretisk både bo med EPS or andre voksne samtidig. Men det blir kanskje noe for dem å finne ut av når de leker med dette ute i test.

Reply:
Det er litt tidlig på morgenen her nå, men er ikke `(delerBolig != null && !delerBolig)` ekvivalent med `delerBolig == false`?